roles/splunk/tasks/configure_license.yml

```diff
@@ -1,4 +1,33 @@
 ---
+- block:
+  - name: Copy Licenses to Master
+    copy:
```

Comment:
Ansible-lint will complain if you don't define ownership and mode. Please add the following 3 lines to your copy:
```
    owner: "{{ splunk_nix_user }}"
    group: "{{ splunk_nix_group }}"
    mode: 0644
```